packages/ngtools/webpack/package.json

```diff
@@ -29,13 +29,13 @@
   },
   "peerDependencies": {
     "@angular/compiler-cli": "^8.0.0-beta.0 || ^8.1.0-beta.0 || ^8.2.0-beta.0 || ^8.3.0-beta.0 || ^8.4.0-beta.0 || >=9.0.0-beta < 9",
-    "typescript": ">=3.4 < 3.6",
+    "typescript": ">=3.4 < 3.7",
```

Comment:
Maybe we should update the peerDeps to only support Angular 9 and TypeScript 3.6?

Reply:
Done